### ISSUE TITLE
Cross-Origin Request Blocked on publisher settings page

### ISSUE THREAD

**Issue by mikeavena** [superdesk/superdesk-publisher#211]
Cross-origin requests are blocked by my Publisher instance, and Superdesk cannot load Publisher settings. I've tried setting CORS_ALLOW_ORIGIN in my publisher instance's .env to '*', '<_superdesk_domain_>', and 'null' with no difference. The following errors are output to the console:

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://<_publisher_domain_>/api/v2/auth/superdesk/. (Reason: CORS header ‘Access-Control-Allow-Origin’ does not match ‘null’).`

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://<publisher_domain>/api/v2/auth/superdesk/. (Reason: CORS request did not succeed).`

`Possibly unhandled rejection: {"data":null,"status":-1,"config":{"method":"POST","transformRequest":[null],"transformResponse":[null],"jsonpCallbackParam":"callback","url":"http://<publisher_domain>/api/v2/auth/superdesk/","data":{"session_id":"<session id>","token":"Basic <token removed>"},"add_format":"json","headers":{"Authorization":"Basic undefined","Accept":"application/json, text/plain, */*","Content-Type":"application/json;charset=utf-8"}},"statusText":"","xhrStatus":"error"}`

**Comment by mikeavena** [comment]
For anyone else having this issue, I was able to fix these console errors by allowing CORS on the Publisher server using Apache's configuration.  I had to load the headers module in apache2.conf with `LoadModule headers_module modules/mod_headers.so` then add `Header set Access-Control-Allow-Origin <http://example.com>`

However, Superdesk still does not connect to the Publisher instance. I only see the three dot loading animation on the Publisher Settings page with no console errors. Is there something else I'm missing?
